### DISCUSSION TITLE
Possible error when compiling

### DISCUSSION THREAD

**Discussion by Phoenixx1202** [bucanero/apollo-ps3#59] [Q&A] [answered]
I'm having problems running the compiled application on PS3, it opens and closes by itself, I've tried manually via pendrive and also via ps3loadx, apparently it doesn't show an error when compiling, they are the same tools I used to compile pkgi (this one it worked out)

**Comment by bucanero** [comment, marked as answer]
Apollo needs the latest ps3 toolchain and some updated libraries to work, so it really depends on your environment.
Check #47 where Crystal shared some notes about building Apollo with the latest psl1ght toolchain.

You can also get a compatible pre-compiled toolchain from here:
https://github.com/bucanero/ps3toolchain/releases

**Reply by Phoenixx1202** [reply]
You're the best, it worked really well here, thanks for the feedback, ps I'm your fan.

Thanks very much
